File: common/src/main/java/net/impleri/playerskills/integration/kubejs/events/EventsBinding.java
package net.impleri.playerskills.integration.kubejs.events;

public abstract class EventsBinding {

    public static final String REGISTRATION = "skills.registration"; //, () -> SkillsRegistrationEventJS.class);

    public static final String MODIFICATION = "skills.modification"; //, () -> SkillsModificationEventJS.class);

    public static final String SKILL_CHANGED = "skills.onChanged"; //, () -> PlayerSkillChangedEventJS.class).extra(Extra.ID);
}

File: common/src/main/kotlin/net/impleri/playerskills/integrations/kubejs/events/EventsBinding.kt
package net.impleri.playerskills.integrations.kubejs.events

import dev.latvian.mods.kubejs.event.EventGroup
import dev.latvian.mods.kubejs.event.EventHandler
import dev.latvian.mods.kubejs.event.Extra

object EventsBinding {
  val GROUP: EventGroup = EventGroup.of("SkillEvents")
  val REGISTRATION: EventHandler = GROUP.startup("registration") { SkillsRegistrationEventJS::class.java }
  val MODIFICATION: EventHandler = GROUP.server("modification") { SkillsModificationEventJS::class.java }
  val SKILL_CHANGED: EventHandler = GROUP.server("onChanged") { PlayerSkillChangedEventJS::class.java }
    .extra(Extra.ID)
}
